Tests/DeepLearningTests/LayerTests.swift

```diff
@@ -66,6 +66,15 @@ final class LayerTests: XCTestCase {
         XCTAssertEqual(output, expected)
     }
 
+    func TestUpSampling3D(){
```

Comment:
Minor: please use camelcase and add a space.
```suggestion
    func testUpSampling3D() {
```

Reply:
@dan-zheng Is the logic right? I'm sorta unsure about the assertion i'm doing.

Reply:
should `expected = TensorShape([1, shape[0]* size, ., . ,1])`